packages/react/src/components/ToggleSmall/ToggleSmall-story.js

```diff
@@ -62,3 +63,38 @@ Default.parameters = {
       `,
   },
 };
+
+export const Skeleton = () => {
```

Comment:
Since we've already deprecated this component, I don't think we need to add in features. Can we just add a `size` prop to the regular `Toggle` variant if users want a smaller Toggle skeleton? I'd prefer to not add anything to `ToggleSmall` since we'll be removing it completely in a few months 👍🏻

Reply:
I didn't add any new features to the `ToggleSmallSkeleton`, I just added the story for it. I fixed style errors that it had, bc wasn't sure if I should still fix the styles for it, but otherwise no new props or anything for this one.

Reply:
Should I just delete the skeleton styles for it then? I wasn't sure if I should fix the `ToggleSmallSkeleton` or not? With the style updates I made to `ToggleSkeleton` it completely breaks the `ToggleSmallSkeleton`, that's why I added the new class, so that it could be temporarily fixed until it's removed all together. Wasn't sure how I should go about this.

Reply:
I just tried rendering the `ToggleSmallSkeleton` on the `main` branch and it seems like it's already broken, so no sweat 😄   I'm down either way. If we do keep it, can you just add a `V11: ` somewhere in the comment so that we can easily remember what we need to remove down the line? 

Also, I think we'll need to remove pointer-events when it is a skeleton 

![2021-04-23 09 16 31](https://user-images.githubusercontent.com/11928039/115900417-ca951080-a414-11eb-9b95-21cc2b3b4c55.gif)